modules/va_facilities/lib/va_facilities/param_validators.rb

```diff
@@ -34,6 +34,31 @@ def validate_type_and_services_known
       raise Common::Exceptions::InvalidFieldValue.new('services', unknown) unless unknown.empty?
     end
 
+    def validate_street_address
```

Comment:
We have been using `ParamValidators` for shared validation methods and leaving validation used by just one controller in that controller. If we are going to move all validators into here, I would suggest pulling them out of `FacilitiyController` as well.

Reply:
oh ok, yea I wasn't sure why it was there in the first place since other validators were in that class.  I'm happy to move all the facility related param validators to the param_validators class if that's fine with you.